Makefile

```diff
@@ -25,14 +25,14 @@ run_test:
 	analysis-runner \
 	--dataset seqr \
 	--access-level test \
-	--output-dir "gs://cpg-seqr-test-tmp/seqr_$(VERSION)/hail" \
-	--description "test seqr loader - batch small1" \
+	--output-dir  "gs://cpg-seqr-test-tmp/hail" \
+	--description "test seqr loader - test" \
 	batch_seqr_loader/batch_workflow.py \
-	--gvcf-bucket "gs://cpg-seqr-test/gvcf/small1" \
-	--cram-bucket "gs://cpg-seqr-test/cram/small1" \
-	--ped-file    "gs://cpg-seqr-test/gvcf/small1/samples.ped" \
-	--data-bucket "gs://cpg-seqr-test/data/test-$(VERSION)" \
-	--work-bucket "gs://cpg-seqr-test-tmp/work/loader-$(VERSION)/work-small1-cram" \
+	--namespace   test \
+	--version     $(TEST_VERSION) \
+	--dataset     "BB01-BB02" \
```

Comment:
It's a bit confusing that we have two different `--dataset` parameters here, which mean different things:

- `seqr` is the dataset in the "stack" sense.
- `BB01-BB02` is the dataset within the seqr stack.

Can we call the latter `seqr-dataset` or `callset`, to make the distinction more clear?

Reply:
Yeah, good point!